aws/nextflow-policies.tf

```diff
@@ -0,0 +1,173 @@
+# S3 Group policies taken from AWS Nextflow batch setup
+resource "aws_iam_policy" "nextflow_readwrite_S3" {
+  name   = "nextflow-ccdl-readwrite-s3"
+  policy = <<EOF
+{
+ "Version": "2012-10-17",
+ "Statement": [
+  {
+   "Effect": "Allow",
+   "Action": [
+    "s3:PutAnalyticsConfiguration",
+    "s3:GetObjectVersionTagging",
+    "s3:ReplicateObject",
+    "s3:GetObjectAcl",
+    "s3:GetBucketObjectLockConfiguration",
+    "s3:PutLifecycleConfiguration",
+    "s3:GetObjectVersionAcl",
+    "s3:PutObjectTagging",
+    "s3:DeleteObject",
+    "s3:DeleteObjectTagging",
+    "s3:GetBucketPolicyStatus",
+    "s3:GetObjectRetention",
+    "s3:GetBucketWebsite",
+    "s3:PutReplicationConfiguration",
+    "s3:DeleteObjectVersionTagging",
+    "s3:PutObjectLegalHold",
+    "s3:GetObjectLegalHold",
+    "s3:GetBucketNotification",
+    "s3:PutBucketCORS",
+    "s3:GetReplicationConfiguration",
+    "s3:ListMultipartUploadParts",
+    "s3:PutObject",
+    "s3:GetObject",
+    "s3:PutBucketNotification",
+    "s3:PutBucketLogging",
+    "s3:GetAnalyticsConfiguration",
+    "s3:PutBucketObjectLockConfiguration",
+    "s3:GetObjectVersionForReplication",
+    "s3:GetLifecycleConfiguration",
+    "s3:GetInventoryConfiguration",
+    "s3:GetBucketTagging",
+    "s3:PutAccelerateConfiguration",
+    "s3:DeleteObjectVersion",
+    "s3:GetBucketLogging",
+    "s3:ListBucketVersions",
+    "s3:ReplicateTags",
+    "s3:RestoreObject",
+    "s3:ListBucket",
+    "s3:GetAccelerateConfiguration",
+    "s3:GetBucketPolicy",
+    "s3:PutEncryptionConfiguration",
+    "s3:GetEncryptionConfiguration",
+    "s3:GetObjectVersionTorrent",
+    "s3:AbortMultipartUpload",
+    "s3:PutBucketTagging",
+    "s3:GetBucketRequestPayment",
+    "s3:GetObjectTagging",
+    "s3:GetMetricsConfiguration",
+    "s3:PutBucketVersioning",
+    "s3:GetBucketPublicAccessBlock",
+    "s3:ListBucketMultipartUploads",
+    "s3:PutMetricsConfiguration",
+    "s3:PutObjectVersionTagging",
+    "s3:GetBucketVersioning",
+    "s3:GetBucketAcl",
+    "s3:PutInventoryConfiguration",
+    "s3:GetObjectTorrent",
+    "s3:PutBucketWebsite",
+    "s3:PutBucketRequestPayment",
+    "s3:PutObjectRetention",
+    "s3:GetBucketCORS",
+    "s3:GetBucketLocation",
+    "s3:ReplicateDelete",
+    "s3:GetObjectVersion"
```

Comment:
Is this everything? I think `s3:*` might be equivalent.

Reply:
I don't know? Probably is nearly equivalent, but this was one that I copied from the CloudFormation setting, so I didn't want to mess around too much. I doubt changing it to s3:* would be a problem.

Reply:
Word if you got this from there then it's probably best to leave it as is.